zeekscript/formatter.py

```diff
@@ -561,15 +561,25 @@ def format(self):
         self._format_child()  # <expr>
 
 
-class EnumBodyFormatterMixin(ComplexSequenceFormatterMixin):
```

Comment:
So glad this is gone, the way it was passed into ``TypeFormatter` felt always weird to me. ❤️